rfcs/text/0042/risk.yml

```diff
@@ -0,0 +1,168 @@
+
+        Risk Categories logically group risk inputs from various domain use cases - e.g. Alert, Posture Management, Vulnerabilities, and others.
+    - name: category_2_count
+      level: extended
+      type: long
+      example: 1921
+      description: >
+        The number of risk input documents that contributed to the Category 2 score.
+
+        Risk Categories logically group risk inputs from various domain use cases - e.g. Alert, Posture Management, Vulnerabilities, and others.
+    - name: category_3_score
+      level: extended
+      type: float
+      example: 25.0
+      description: >
+        The contribution of Category 3 to the overall risk score (`calculated_score`).
+
+        Category 3 includes inputs from Vulnerabilities.
+
+        Risk Categories logically group risk inputs from various domain use cases - e.g. Alert, Posture Management, Vulnerabilities, and others.
+    - name: category_3_count
+      level: extended
+      type: long
+      example: 4920
+      description: >
+        The number of risk input documents that contributed to the Category 3
+        score.
+
+        Risk Categories logically group risk inputs from various domain use cases - e.g. Alert, Posture Management, Vulnerabilities, and others.
+    - name: category_4_score
+      level: extended
+      type: float
+      example: 55.0
+      description: >
+        The contribution of Category 4 to the overall risk score (`calculated_score`).
+
+        Category 4 contains Entity Contexts.
+
+        Risk Categories logically group risk inputs from various domain use cases - e.g. Alert, Posture Management, Vulnerabilities, and others.
+    - name: category_4_count
+      level: extended
+      type: long
+      example: 1308
+      description: >
+        The number of risk input documents that contributed to the Category 4 score.
+
+        Risk Categories logically group risk inputs from various domain use cases - e.g. Alert, Posture Management, Vulnerabilities, and others.
+    - name: category_5_score
+      level: extended
+      type: float
+      example: 75.0
+      description: >
+        The contribution of Category 5 to the overall risk score (`calculated_score`).
+
+        Category 5 contains inputs from Anomalies.
+
+        Risk Categories logically group risk inputs from various domain use cases - e.g. Alert, Posture Management, Vulnerabilities, and others.
+    - name: category_5_count
+      level: extended
+      type: long
+      example: 14922
+      description: >
+        The number of risk input documents that contributed to the Category 5 score.
+
+        Risk Categories logically group risk inputs from various domain use cases - e.g. Alert, Posture Management, Vulnerabilities, and others.
+    - name: risk_inputs
+      level: extended
+      type: object
+      normalize: array
+      description: >
+        An array of the top risk input documents that contributed to the overall risk score.
+    - name: risk_inputs.id
```

Comment:
IMO feels redundant to have fields named `risk.risk_inputs.*` (with `risk_` prepended).

Reply:
I moved forward on the assumption that they are equivalent, and then did that nesting. Let me know if that's inappropriate (either for the declaration, or for ECS tooling, etc.)